### DISCUSSION TITLE
Installing via Laravel

### DISCUSSION THREAD

**Discussion by lenholmsthlm** [tailwindlabs/tailwindcss#3803] [Help]
Hey so I'm new to this and I'm following a Youtube tutorial I've also read the official docs and have done every step but I still get an error when I try to load the tailwind. 

So my webpack looks like this
```
 mix.js("resources/js/app.js", "public/js")
 .postCss("resources/css/app.css", "public/css", [
    require("tailwindcss"),

 ]);
```

And when I go to app.css and import these commands
@tailwind base;
@tailwind components;
@tailwind utilities;

It gives me an error saying 
 "Unknown at rule @tailwindcss(unknownAtRules)"

I don't know what to do have been stuck on this for awhile I've tried reinstalling but dosen't help. I've done everything that's mentioned here https://tailwindcss.com/docs/guides/laravel

Thanks in advance

**Comment by simonswiss** [comment]
Hello!

Are you able to share a GitHub repo of your installation setup, so we can have a look?
